clockify.go

```diff
@@ -135,6 +135,21 @@ func (c *Clockify) Workspaces() ([]Workspace, error) {
 	return workspaces, nil
 }
 
+func (c *Clockify) FindWorkspace(workspaces []Workspace, name string) (Workspace, error) {
```

Comment:
Looks like this function can stand alone:
```suggestion
func selectWorkspace(workspaces []Workspace, name string) (Workspace, error) {
```